### PR TITLE
zoom control 200px wide to increase accuracy

### PR DESCRIPTION
**Pull request by ctzsnooze**
While working with GPS Rescue graphs, it's great to zoom in a lot, eg 1%, 2%, 3%.

With the zoom element width of 130px, it was not possible (on my mac, anyway) to get 2% zoom.

By increasing the element width in CSS to 200px, it's a lot easier.

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by haslinghuis — APPROVED** [review]

**Pull request merged**
